### PR TITLE
Fixed #697 -- Corrected url configuration for profile view

### LINKED ISSUE #697: Profile view urlconf doesn't include all valid characters
The profile view URLConf matches usernames with `[\w-]+` but usernames can actually contain `[\w.@+-]+`.

Changing the urlconf to include the extra characters should fix this issue.

### PR DESCRIPTION
**Pull request by bmispelon**

### CODE REVIEW

File: accounts/tests.py [hunk 1]
```diff
@@ -15,3 +15,11 @@ def test_user_profile(self, mock_user_stats):
         response = self.client.get(reverse('user_profile', host='www', args=['a-user']))
         self.assertContains(response, 'a-user')
         mock_user_stats.assert_called_once_with(self.user)
+
+    def test_profile_view_reversal(self):
+        """
+        Make sure the profile view can be reversed for usernames containing
```
Comment (timgraham) [inline accounts/tests.py]: Chop "Make sure" prefix since all tests make sure of stuff.

**Comment by timgraham** [comment]
LGTM, how about "URLconf" in the commit message instead of "url configuration".

p.s. you created a conflict with my PR.  :boom: :truck:

**Comment by bmispelon** [comment]
@timgraham Fixed the two comments you had, thanks.

I'm happy to wait until you merge yours and fix the conflicts afterwards. 😅

**Comment by timgraham** [comment]
Can you :mag: my PR?

**Comment by bmispelon** [comment]
Rebased and resolved conflicts.

**Pull request merged**